src/mailbox.cpp

```diff
@@ -80,35 +94,35 @@ bool Mailbox::sendItem(Item* item) const
 		return false;
 	}
 
-	Player* player = g_game.getPlayerByName(receiver);
+	auto player = g_game.getPlayerByName(receiver);
```

Comment:
Could this be `const auto`
```suggestion
	const auto player = g_game.getPlayerByName(receiver);
```